tests/integration/Core/Content/Media/Message/UpdateThumbnailsMessageTest.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Integration\Core\Content\Media\Message;
+
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Content\Media\Message\UpdateThumbnailsMessage;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelTestBehaviour;
+
+class UpdateThumbnailsMessageTest extends TestCase
+{
+    use KernelTestBehaviour;
+
+    protected function setUp(): void
+    {
+        $this->serializer = static::getContainer()->get('serializer');
+    }
+
+    public function testDeserialization(): void
+    {
+        $message = new UpdateThumbnailsMessage();
+        $message->setStrict(true);
+
+        $serialized = $this->serializer->serialize($message, 'json');
+        /** @var UpdateThumbnailsMessage $deserialized */
```

Comment:
please do not use var annotations. Especially in tests you can simply assert the type. also see https://github.com/shopware/shopware/blob/trunk/coding-guidelines/core/writing-code-for-static-analysis.md

Reply:
Fixed